financialaid/factories.py

```diff
@@ -0,0 +1,63 @@
+"""
+Factories for financialaid tests
+"""
+import datetime
+from factory import SubFactory
+
+from factory.django import DjangoModelFactory
+from factory.fuzzy import (
+    FuzzyAttribute,
+    FuzzyInteger,
+    FuzzyText,
+    FuzzyChoice, FuzzyFloat, FuzzyDateTime)
+import faker
+from pytz import UTC
+
+from courses.factories import ProgramFactory
+from financialaid.models import TierProgram, Tier, FinancialAidStatus, FinancialAid
+from profiles.factories import UserFactory
+
+
+FAKE = faker.Factory.create()
+
+
+class TierFactory(DjangoModelFactory):
+    """
+    Factory for Tier
+    """
+    name = FuzzyText()
+    description = FuzzyText()
+
+    class Meta:  # pylint: disable=missing-docstring, no-init, too-few-public-methods, old-style-class
```

Comment:
is it really failing the pylint for `missing-docstring, no-init, too-few-public-methods, old-style-class` ?